Twig/Extension/FormatterMediaExtension.php

```diff
@@ -76,14 +76,6 @@ public function getTwigExtension()
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function getName()
```

Comment:
The method is deprecated, but `getName` still exists in the interface https://github.com/twigphp/Twig/blob/v1.28.0/lib/Twig/ExtensionInterface.php#L86